airrohr-firmware/mhz19.h

```diff
@@ -17,6 +17,7 @@
  ***********************************************************************/
 
 #include <Arduino.h>
+#define MHZ19_SUPPORT_RANGE
```

Comment:
isn't that supposed to be

#undef MHZ19_SUPPORT_RANGE 

?

Reply:
it should have been commented, but I compared the code size between the commented and uncommented version and forgot to change it back...

nevertheless in the next commit eda92fc97b06e5c65c9d69e6ad1cc46096ab22b7 it is already correct.